support/scheduler-client/client.go

```diff
@@ -0,0 +1,251 @@
+//
+// Copyright (c) 2017 Tencent
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package scheduler
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/edgexfoundry/edgex-go/core/domain/models"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+// Common http const
+const (
+	ContentType        = "Content-Type"
+	ContentTypeJsonVal = "application/json"
+)
+
+const (
+	ScheduleApiPath      = "/api/v1/schedule"
+	ScheduleEventApiPath = "/api/v1/scheduleevent"
+	UrlPattern           = "http://%s:%d%s"
+)
+
+// Struct to represent the scheduler client
+type SchedulerClient struct {
```

Comment:
Please make this an interface and create a SchedulerRestClient type to implement it. Examples can be found in any of the core/clients packages.

Remove the initialization of the SchedulerClient from main.go and put it in init.go, following the pattern of other services.